site.yml

```
@@ -21,460 +21,460 @@ Documents:
+        SEO Description: aws-sdk-dotnet is the official AWS SDK for the .NET Framework. In this recipe we will learn how to use aws-sdk-dotnet with MinIO server.
+        SEO Keywords: AWS SDK for .NET
 
       - How to use Minio's server-side-encryption with aws-cli:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/how-to-use-minio-server-side-encryption-with-aws-cli.md
         Slug: how-to-use-minio-s-server-side-encryption-with-aws-cli
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use MinIO's server-side-encryption with aws-cli - Cookbook/Recipe
+        SEO Description: MinIO supports S3 server-side-encryption with customer provided keys (SSE-C), this describes the use of server-side encryption with AWS Command Line Interface (aws-cli)
+        SEO Keywords: aws-cli, server side encryption
 
       - How to monitor Minio using Prometheus:
         Link: https://github.com/minio/cookbook/blob/master/docs/how-to-monitor-minio-with-prometheus.md
         Slug: how-to-monitor-minio-using-prometheus
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to monitor MinIO using Prometheus - Cookbook/Recipe
+        SEO Description: This document explains how to setup Prometheus and configure it to scrape data from MinIO servers.
+        SEO Keywords: Prometheus
 
   - Related Projects:
       - Generate Let's Encrypt certificate using Certbot for Minio:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/generate-lets-encypt-certificate-using-certbot-for-minio.md
         Slug: generate-let-s-encypt-certificate-using-concert-for-minio
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Using the Let's Encrypt Certbot with Minio
+        SEO Description: In this recipe, we will generate a Let's Encrypt certificate using Certbot. This certificate will then be deployed for use in the Minio server.
+        SEO Keywords: Let's Encrypt, Certbot, Certificate Authority
 
       - How to install Golang?:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/how-to-install-golang.md
         Slug: how-to-install-golang
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Installing Golang
+        SEO Description: The guide covers the basics of installing Golang
+        SEO Keywords: Golang
 
   - Minio Deployment:
       - Minio Deployment Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio/master/docs/orchestration/README.md
         Slug: minio-deployment-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | MinIO Deployment Quickstart Guide
+        SEO Description: This guide covers deployment documentation for - Orchestration platforms, Docker Swarm, Docker Compose, Kubernetes, DC/OS
```

Shouldn't we move the slash as follows:

```suggestion
        SEO Description: This guide covers deployment documentation for Orchestration platforms - Docker Swarm, Docker Compose, Kubernetes, DC/OS
```